### PR TITLE
Removing extra protobuf notice file

### LINKED ISSUE #9287: [BUG] Empty protobuf-java-NOTICE.txt file in project root
**Describe the bug**
There is an empty [`protobuf-java-NOTICE.txt`](https://github.com/opensearch-project/OpenSearch/blob/main/protobuf-java-NOTICE.txt) file in the project root, committed in https://github.com/opensearch-project/OpenSearch/pull/6960.  I suspect one or more of the following conditions is true:
 - The file was auto-generated at some point and accidentally committed, and should be deleted/committed and then added to `.gitignore`
 - The file has content that should be there that was deleted
 - There is missing content in [NOTICE.txt](https://github.com/opensearch-project/OpenSearch/blob/main/NOTICE.txt).

**Expected behavior**

Any license requirements are followed properly.

Any file named `NOTICE` should have some content explaining its existence.

### PR DESCRIPTION
**Pull request by VachaShah**
### Description
Same as title

### Related Issues
Resolves #9287 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff
- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kotwanikunal — APPROVED** [review]

**Review by Poojita-Raj — APPROVED** [review]

File: plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java [hunk 1]
```diff
@@ -191,7 +191,7 @@ public void testTypeParser_parse_fromCorrelationParamsContext_InvalidVectorSimil
 
     /**
      * test parseCreateField in CorrelationVectorFieldMapper
-     * @throws IOException
```
Comment (owaiskazi19) [inline plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java:194]: Can we add more details for the IOException here?

File: plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java [hunk 2]
```diff
@@ -191,7 +191,7 @@ public void testTypeParser_parse_fromCorrelationParamsContext_InvalidVectorSimil
...
+     * @throws IOException ioexception
```
Comment (owaiskazi19) [inline plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java:194]: Can we add more details for the IOException here? Also, I think the PR title is different than the changes in this PR?

Comment (VachaShah) [reply plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java:194]: I am removing the extra protobuf-java file. The others came up as errors on my local precommit.

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25876/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

Flaky test reported in #10006

**Review by owaiskazi19 — APPROVED** [review]

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25879/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

Flaky test reported in https://github.com/opensearch-project/OpenSearch/issues/10133

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25929/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

More flaky tests reported in https://github.com/opensearch-project/OpenSearch/issues/10006, https://github.com/opensearch-project/OpenSearch/issues/8887

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25940/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

Flaky test reported in https://github.com/opensearch-project/OpenSearch/issues/10133

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:**  ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25959/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

Flaky tests in https://github.com/opensearch-project/OpenSearch/issues/7357 and https://github.com/opensearch-project/OpenSearch/issues/10133

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/26026/
> * **CommitID:** [e97b051](https://github.com/opensearch-project/OpenSearch/commit/e97b051ccdf76e32b8914f7b8ef63236b4340d13)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

Flaky test again https://github.com/opensearch-project/OpenSearch/issues/10006

**Comment by kotwanikunal** [comment]
Ugh. This gradle check history is bad. :(
@VachaShah can you try rebasing instead? That might fix your problems :)

**Comment by VachaShah** [comment]
> Ugh. This gradle check history is bad. :( @VachaShah can you try rebasing instead? That might fix your problems :)

Yeah :( I just force pushed with a rebase. Thank you!

File: plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java (hunk 2 above)
Comment (andrross) [reply plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java:194]: @VachaShah Just FYI tests should be excluded from javadoc enforcement. I've seen cases where this doesn't work properly if building with JDK-14, so if that's the case for you I'd suggest moving to JDK 11 or 17.

**Comment by VachaShah** [comment]
> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/26032/
> * **CommitID:** [34cbc6b](https://github.com/opensearch-project/OpenSearch/commit/34cbc6bd0573dc67202591ad05102df37c6a07af)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

testPrimaryTermValidation failing again. https://github.com/opensearch-project/OpenSearch/issues/10133

**Comment by VachaShah** [comment]
@kotwanikunal Any suggestions to get a clean gradle check? :(

**Comment by kotwanikunal** [comment]
> @kotwanikunal Any suggestions to get a clean gradle check? :(

I wish there was another way out. I re-ran it 🤞

**Comment by kotwanikunal** [comment]
@VachaShah 🎊

File: plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java (hunk 2 above)
Comment (VachaShah) [reply plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java:194]: Ah thanks @andrross, that might be the case why it failed for me.

**Comment by VachaShah** [comment]
> @VachaShah 🎊

Finally 🎊  Thanks @kotwanikunal!

**Pull request merged**